src/Utilities.h

```diff
@@ -75,14 +75,14 @@ bool directoryExists(const std::string& directory);
  * @return Vector of summation values
  */
 template<typename T>
-std::vector<T> AccumulateEveryN(const std::vector<T>& data, int n)
+std::vector<T> AccumulateEveryN(const std::vector<T>& data, unsigned n)
```

Comment:
I think it's better to explicitly say `unsigned int n`, rather than just `unsigned n`, for consistency with the rest of the code.